matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/registration/RegistrationWizard.kt

```diff
@@ -119,8 +121,8 @@ interface RegistrationWizard {
     val isRegistrationStarted: Boolean
 
     /**
-     * True when login and password have been sent with success to the homeserver, i.e. [createAccount] has been
-     * called successfully.
+     * This is a session identifier that the client must pass back to the homeserver,
```

Comment:
It confuses the SDK documentation reader.

Maybe tell that the value is exposed, but it is not necessary that the application uses it to be able to create an account.

This is why I would like it to stay hidden from the SDK API.